### ISSUE TITLE
Long content in table cell resulting in error: ! Undefined control sequence.

### ISSUE THREAD

**Issue by Muminologist** [Zettlr/Zettlr#1585]
## Description

PDF-export of a table with pandocc-2.11.3.2.big_sur exits with error message:
> Error producing PDF.
> ! Undefined control sequence. 

## Reproducing

Export a markdown document from Zettlr that contains a table with at least one cell having a bit longish content. 

Minimal working example:
| This | is a                                                                            | minimal working example |
|-----------------|----------------------------------------------------------------------------------|-------------|
| 1               | short line | 35-60       |
|                 |                                                                                  |             |
| 2               | This is a really, really long text that requires a kind of linebreak or wrapping within the cell       | 61-194      |

## Expected behaviour

Should generate proper pdf with nicely wrapped content in the particular cell.

Error Message:
```
Error producing PDF.
! Undefined control sequence.
<argument> @{} >{\raggedright \arraybackslash 
                                              }p{(\columnwidth - 4\tabcolsep...
l.132 ...mnwidth - 4\tabcolsep) * \real{0.12}}@{}}
```

## Platform
 - OS and version: MacOS Big Sur 11.0.1
 - Zettlr Version: Zettlr-1.8.4
 - Pandoc Version: 2.11.3.2.big_sur
 - MacTex Version: 20200407

## Additional information

Generating a LibreOffice or Word file from the same source works perfectly fine - as does general PDF creation.

**Comment by nathanlesage** [comment]
It almost looks as if pandoc would then fall back to some different commands which are not present in Zettlr's template, could this be? @Kangie

**Comment by Muminologist** [comment]
Thanks @nathanlesage . 
Let me add some additional info. The file in question worked perfectly fine before. I have a Zettlr created pdf of this document from 12/16/2020 (probably 1.8.0). Also, using standard pandoc to process said file works flawless.

**Comment by Kangie** [comment]
Yeah the array backslash is required for the current version of pandoc. We've already merged a fix for this into the template (#1566), but I don't think that there's been a release since.

**Comment by nathanlesage** [comment]
> Yeah the array backslash is required for the current version of pandoc. We've already merged a fix for this into the template (#1566), but I don't think that there's been a release since.

Aaaaah, see? This is how much I have an overview over the app right now :D

I'm so happy once we finally have Pandoc coupled to the app, and I can finally throw away all that Pandoc command nightmare.

Closing as it'll be fixed in 1.8.5!
